HLTrigger/HLTfilters/plugins/plugins.cc

```diff
@@ -127,6 +127,10 @@ typedef HLTDoublet<RecoChargedCandidate, PFMET> HLT2MuonPFMET;
 typedef HLTDoublet<RecoEcalCandidate, MET> HLT2PhotonMET;
 typedef HLTDoublet<RecoEcalCandidate, PFMET> HLT2PhotonPFMET;
 
+#include "HLTDoubletSinglet.h"
+#include "HLTDoubletSinglet.cc"
```

Comment:
In CMSSW we try to avoid implementation in header files, so I keep it as it is.

Reply:
I simply read [here](https://cms-sw.github.io/cms_coding_rules.html#4--technical-coding-rules-1) ("4.3 Header ﬁles must not contain any implementation except for class templates and code to be inlined."); this is a class template and i couldn't see the advantage of having the `.cc` file. In any case, I won't insist, since other plugins in this pkg follow this pattern.

Reply:
Yes, right, it is template class.

Reply:
The `.cc` file merged with `.h` as suggested